.travis.yml

```diff
@@ -27,6 +34,9 @@ script: |
     mvn clean verify --settings maven-settings.xml -B -V -Dgpg.skip
   fi
 
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
[cringe]

Reply:
I asked a question about this on stackoverflow: 
 https://stackoverflow.com/questions/46308469/security-concerns-about-running-scripts-on-travis-ci

Reply:
Good idea to ask for input on Stack Overflow. It seemed to me that the curl-pipe-bash method was born out of a mixture of laziness and naivete, but would be interested in getting alternate perspectives. The comments on this one sum up the usual reaction I've seen to such things ("horrible", "inadmissible" etc.) https://gist.github.com/btm/6700524